windows_containers/index.adoc

```diff
@@ -0,0 +1,33 @@
+[id="windows-container-overview"]
+= Windows container overview
+include::modules/common-attributes.adoc[]
+:context: windows-container-overview
+
+toc::[]
+
+Windows Container Support for {product-title} is a feature providing the ability to run Windows compute nodes in an {product-title} cluster. This is possible by using the Red Hat Windows Machine Config Operator (WMCO) to install and manage Windows nodes. By using Red Hat subscription, you can get support for Windows Container. For more information see, xref:../windows_containers/windows-containers-release-notes-4-x.adoc#windows-containers-release-notes-4-x[Release notes].
+
+xref:../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Windows container workloads]:
+For workloads including both Linux and Windows, {product-title} allows you to deploy Windows workloads running on Windows Server containers while also providing traditional Linux workloads hosted on Red Hat Enterprise Linux CoreOS (RHCOS) or Red Hat Enterprise Linux (RHEL).
+Windows container workloads support for clusters running on the following cloud providers:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+
+xref:../windows_containers/enabling-windows-container-workloads.adoc#enabling-windows-container-workloads[Enable Windows container workloads]: You need a Machine Config Operator (WMCO) to run Windows workloads in your cluster. The WMCO orchestrates the process of deploying and managing Windows workloads on a cluster.
+
+Creating a Windows MachineSet object: You can create a Windows MachineSet object to create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines. You can create a Windows MachineSet object on the following platforms:
+
+* xref:../windows_containers/creating-windows-machinesets/./creating-windows-machineset-aws.adoc#creating-windows-machineset-aws[AWS]
+* xref:../windows_containers/creating-windows-machinesets/./creating-windows-machineset-azure.adoc#creating-windows-machineset-azure[Azure]
```

Comment:
I'm not certain listing providers we can create a MachineSet for is all that relevant. How PM/engineering has broken down platform-support with the WMCO is outlined [here](https://docs.openshift.com/container-platform/4.9/windows_containers/understanding-windows-container-workloads.html#supported-platforms-based-on-openshift-container-platform-and-wmco-versions). However, I think it would be wise to simply link to that section instead of repeating the info from it to cut down on places to update as that info changes. :slightly_smiling_face:

Reply:
acked. Removed the listing and now I have mentioned as- `You can create a Windows MachineSet object on the multiple platforms.` Is that accurate? Let me know if any other changes are required.
There is a link available only to subpoints only so couldn't add the link for the whole section.

Reply:
Yeah, I think that works. With the platform support changing frequently, better to just term it generically and let the prereqs module handle the specifics.

Reply:
ack